build/dist

```diff
@@ -136,7 +141,11 @@ HIVE_VERSION=$("$MVN" help:evaluate -Dexpression=hive.version $@ 2>/dev/null\
 echo "Building Kyuubi package of version $VERSION against Spark version - $SPARK_VERSION"
 
 if [[ "$NAME" == "none" ]]; then
-  NAME="spark-"$SPARK_VERSION
+  if [[ "$SPARK_PROVIDED" == "true" ]]; then
+    NAME="without-spark"
+  else
+    NAME="spark-"$SPARK_VERSION
```

Comment:
Can we change it to use `major.minor` version form?`, also cc @ulysses-you WDYT?

Reply:
I'm ok with this change, but it better change in separate PR.